app/javascript/gobierto_dashboards/webapp/containers/tender/TendersShow.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div>
-    <h1>{{ formattedTender.title }}</h1>
+    <h1>{{ tender.title }}</h1>
```

Comment:
Some concept as Contracts. Is actually so different this component from the contracts one that we need another component?

Reply:
Tenders page may not be in the final versión. If it finally is, I'll merge the logic.